tutorials/Numerics/DGMethods/box1D.jl

```diff
@@ -0,0 +1,301 @@
+        "Box1D",
+        N_poly,
+        nelem,
+        zmax,
+        param_set,
+        m,
+        numerical_flux_first_order = CentralNumericalFluxFirstOrder(),
+        boundary = ((0, 0), (0, 0), (0, 0)),
+        periodicity = (true, true, true)
+    );
+
+    t0 = FT(0)
+    timeend = FT(370)
+
+    Δ = min_node_distance(driver_config.grid, VerticalDirection())
+    max_vel = m.velo
+    dt = Δ / max_vel
+
+    solver_config =
+        ClimateMachine.SolverConfiguration(t0, timeend, driver_config, ode_dt = dt);
+    grid = solver_config.dg.grid;
+    Q = solver_config.Q;
+    aux = solver_config.dg.state_auxiliary;
+
+    output_dir = @__DIR__;
+    mkpath(output_dir);
+    z_key = "z"
+    z_label = "z"
+    z = get_z(grid)
+    state_vars = SingleStackUtils.get_vars_from_nodal_stack(
+        grid,
+        Q,
+        vars_state_conservative(m, FT),
+    )
+    aux_vars = SingleStackUtils.get_vars_from_nodal_stack(
+        grid,
+        aux,
+        vars_state_auxiliary(m, FT),
+    )
+    all_vars = OrderedDict(state_vars..., aux_vars...);
+
+    n_outputs = 2
+    all_data = Dict[Dict([k => Dict() for k in 0:n_outputs]...),]
+    all_data[1] = all_vars # store initial condition at ``t=0``
+
+    # output
+    step = [1];
+    output_freq = floor(Int, timeend / dt);
+    @info(" ", timeend, dt, output_freq)
+
+    cb_output = GenericCallbacks.EveryXSimulationSteps(output_freq) do
+        state_vars = SingleStackUtils.get_vars_from_nodal_stack(
+            grid,
+            Q,
+            vars_state_conservative(m, FT),
+        )
+        aux_vars = SingleStackUtils.get_vars_from_nodal_stack(
+            grid,
+            aux,
+            vars_state_auxiliary(m, FT);
+        )
+        all_vars = OrderedDict(state_vars..., aux_vars...)
+        push!(all_data, all_vars)
+        @info("AQQ")
+        step[1] += 1
+        nothing
+    end;
+
+    filter_freq = 1
+    # tmar filter
+    cb_tmar =
+        GenericCallbacks.EveryXSimulationSteps(filter_freq) do (init = false)
+            Filters.apply!(
+                solver_config.Q,
+                (:q,),
+                solver_config.dg.grid,
+                TMARFilter(),
+            )
+            nothing
+        end
+    # cutoff filter
+    cb_cutoff =
+        GenericCallbacks.EveryXSimulationSteps(filter_freq) do (init = false)
+            Filters.apply!(
+                solver_config.Q,
+                (:q,),
+                solver_config.dg.grid,
+                CutoffFilter(solver_config.dg.grid, cutoff_param),
+            )
+            nothing
+        end
+    # exponential filter
+    cb_exp =
+        GenericCallbacks.EveryXSimulationSteps(filter_freq) do (init = false)
+            Filters.apply!(
+                solver_config.Q,
+                (:q,),
+                solver_config.dg.grid,
+                ExponentialFilter(solver_config.dg.grid, exp_param_1, exp_param_2)
+            )
+            nothing
+        end
```

Comment:
Can you add the Boyd-Vandeven filter to this too?

Reply:
@thomasgibson - I'm adding it now. I guess it should also be added to the filter tests in `ClimateMachine.jl/test/Numerics/Mesh/filter.jl`. But I don't know how to test it.